### ISSUE TITLE
Ruby 2.4.1 Release

### ISSUE THREAD

**Issue by jfly** [whitequark/parser#346]
Similar to #344 and #326, we're getting a warning when running Ruby 2.4.1.

```
warning: parser/current is loading parser/ruby24, which recognizes
warning: 2.4.0-compliant syntax, but you are running 2.4.1.
warning: please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
```

**Comment by whitequark** [comment]
See https://github.com/whitequark/parser/pull/342#issuecomment-289304128.

**Comment by bbatsov** [comment]
I'll also mention https://github.com/whitequark/parser/issues/317#issuecomment-304316789

TLDR - we need a volunteer for #342.

**Comment by whitequark** [comment]
Not just #342 but every Ruby commit that went into 2.3 and 2.4 branches...

**Comment by bbatsov** [comment]
I'll post a note in the RuboCop issue tracker regarding this - hopefully it will attract some attention.

**Comment by rubys** [comment]
I know that prior issues asking for the ability to silence the warning have been rejected, but there are use cases where support for the latest Ruby language syntax isn't vital, and yet it is worthwhile to stay current on Ruby versions.

**Comment by whitequark** [comment]
I'm grinding through several months of backlog that accumulated due to health issues. I'm going to fix Parser *really soon*, it's nearing the head of the queue.

**Comment by rhymes** [comment]
@whitequark hope you're feeling better and keep up the great work!

**Comment by whitequark** [comment]
Thanks.

**Comment by garrow** [comment]
#342 was closed as a duplicate of this issue, but that issue also referenced 
#336 "[Bug #13073]" which has an open PR, #356  "Fix parse error when hash contains do blocks" 
commenting here just to link them together.

**Comment by ayanko** [comment]
Any updates about release new gem version?

**Comment by whitequark** [comment]
not yet

**Comment by tisba** [comment]
Also waiting for this. But I'm also wondering if there should be a new issue for supporting 2.4.2 (released about a month ago)…

**Comment by ayanko** [comment]
It's boring message... Let's fix & release :)

**Comment by davidlibrera** [comment]
I suppose that we need to wait for christmas

https://github.com/whitequark/parser/commit/0c4462aaa4360c032b2235cb568a73226c54e2c8

**Comment by whitequark** [comment]
Will be released once all [blocking-2.4](https://github.com/whitequark/parser/issues?q=is%3Aissue+is%3Aopen+label%3Ablocking-2.4) issues are done.

**Comment by whitequark** [comment]
2.5.0.0 has been released.
